software/contrib/master_clock.py

```diff
@@ -0,0 +1,340 @@
+from europi import *
+import machine
+from time import ticks_diff, ticks_ms
+from europi_script import EuroPiScript
+import uasyncio as asyncio
```

Comment:
[required] To fix the test failures replace this line with:
```suggestion
try:
    import uasyncio as asyncio
except ImportError:
    import asyncio
```

Reply:
Done! Thanks for find this error.